pkg/whiteblacklist/whiteblacklist.go

```diff
@@ -11,6 +11,11 @@ type WhiteBlackList struct {
 	isWhiteList bool
 }
 
+type Lister interface {
```

Comment:
I would prefer defining interfaces close to the caller for better decoupling. What do you think?

Reply:
Will move it back, makes sense!